packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -3,15 +3,27 @@
 const pick = require('lodash.pick')
 const platform = require('../../platform')
 const DatadogSpanContext = require('../span_context')
+const NoopSpanContext = require('../../noop/span_context')
 const log = require('../../log')
 
 const traceKey = 'x-datadog-trace-id'
 const spanKey = 'x-datadog-parent-id'
 const originKey = 'x-datadog-origin'
 const samplingKey = 'x-datadog-sampling-priority'
 const baggagePrefix = 'ot-baggage-'
+const b3TraceKey = 'x-b3-traceid'
+const b3TraceExpr = /^([0-9a-f]{16}){1,2}$/i
+const b3SpanKey = 'x-b3-spanid'
+const b3SpanExpr = /^[0-9a-f]{16}$/i
+const b3ParentKey = 'x-b3-parentspanid'
+const b3SampledKey = 'x-b3-sampled'
+const b3FlagsKey = 'x-b3-flags'
+const b3HeaderKey = 'b3'
+const b3HeaderExpr = /^(([0-9a-f]{16}){1,2}-[0-9a-f]{16}-[01d](-[0-9a-f]{16})?|[01d])$/i
```

Comment:
What if the sampling state is not included?

https://github.com/openzipkin/b3-propagation#single-header
> `b3={TraceId}-{SpanId}-{SamplingState}-{ParentSpanId}`, where the last two fields are optional.

Reply:
It is included as `[01d]`.

Reply:
Oh I see what you mean, fixing it now.